website/docs/r/external_network_v2.html.markdown

```diff
@@ -20,7 +20,7 @@ requires at least VCD *10.0+*. It supports both NSX-T and NSX-V backed networks
 
 Supported in provider *v3.0+*.
 
-## Example Usage (NSX-T backed external network)
+## Example Usage (NSX-T Tier 0 router backed external network)
```

Comment:
In headings it's easier to read if we capitalize the keywords:
```suggestion
## Example Usage (NSX-T Tier 0 Router backed External Network)
```

Reply:
Adjusted, It is upper case now.